packages/app-desktop/gui/Sidebar/styles/sidebar-header-container.scss

```
@@ -3,4 +3,10 @@
 	display: flex;
 	flex-direction: row;
 	align-items: center;
-}
\ No newline at end of file
+	height: 30px;
+	min-height: 30px;
+	min-width: 0;
+	width: 100%;
+	overflow: hidden;
+}
+}
```

Stylelint (17.4.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)